go/base/tests/test_go_assign_tagpool.py

```diff
@@ -19,9 +19,9 @@ def setUp(self):
 
     def handle_command(self, tagpool=None, max_keys=None, update=False):
         self.command.handle(**{
-            "email-address": self.user_helper.get_django_user().email,
+            "email_address": self.user_helper.get_django_user().email,
             "tagpool": tagpool,
-            "max-keys": max_keys,
+            "max_keys": max_keys,
             "update": update
```

Comment:
Please add a trailing comma here as well, even though it was already missing before you touched it.